sqlite/ext/comdb2/files.c

```diff
@@ -340,6 +340,12 @@ static int read_dir(const char *dirname, db_file_t **files, int *count, char *fi
             continue;
         }
 
+        if (euidaccess(path, R_OK) == -1) {
```

Comment:
I don't think euidaccess is portable. `access()` is probably fine.

Reply:
You're right -- switched to `access()`.